components/sharing/SocialShareModal.vue

```diff
@@ -5,16 +5,46 @@
     hide-header-close
     hide-footer
     data-qa="share modal"
+    v-on="onItemPage && !oEmbedDataHtml?
+      { hide: resetEmbedCopied, show: requestEmbed}
+      :onItemPage ? { hide: resetEmbedCopied }
+        :{}"
   >
     <div class="icon-wrapper">
       <SocialShare
         :media-url="mediaUrl"
         with-text
       />
     </div>
+    <b-form
+      v-if="oEmbedDataHtml"
+      data-qa="share modal embed"
+      class="mt-3"
+      @submit.stop.prevent="submitForm"
+    >
+      <label> {{ $t('record.copyEmbedLabel') }}</label>
```

Comment:
The label element needs to be linked to the textarea element it labels, with the attribute `for="shareEmbed"`.

Also remove the extraneous space after the opening label tag.